app.rb

```diff
@@ -12,54 +12,76 @@ def initialize
   end
 
   def list_books
-    @books.each do |book|
-      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    if @books.empty?
+      puts
+      puts 'No books found'
+    else
+      @books.each_with_index { |book, index| puts "#{index}) Title: #{book.title}, Author: #{book.author}" }
     end
   end
 
   def list_people
-    @people.each do |person|
-      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, AGE: #{person.age}"
+    if @people.empty?
+      puts
+      puts 'No people found'
+    else
+      @people.each_with_index do |person, index|
+        puts "#{index})- [#{person.class}] Age: #{person.age}, Name: #{person.name}, ID: #{person.id}"
+      end
     end
   end
 
   def list_rentals
-    print 'ID of person: '
-    person_id = gets.chomp.to_i
-    puts 'Rentals:'
-    @rentals.each do |rental|
-      unless rental.person.id != person_id
-        puts "Date: #{rental.date}, Book \"#{rental.book.title}\" by #{rental.book.author}"
+    if @rentals.empty?
+      puts 'No rental mode'
+    else
+      print 'ID of person: '
+      person_id = gets.chomp.to_i
+      person_rentals = @rentals.select { |rental| rental.person.id == person_id }
+      if person_rentals.empty?
+        puts 'Person ID not found'
+      else
+        person_rentals.each do |list|
+          puts "Date: #{list.date}, Books: #{list.book.title} written by Author: #{list.book.author}"
+        end
       end
     end
   end
 
+  def create_student
+    print 'Age: '
+    person_age = gets.chomp.to_i
+    print 'Name: '
+    person_name = gets.chomp.to_s
+    print 'Has parent permission? [Y/N]: '
+    person_permission = gets.chomp.to_s.capitalize
```

Comment:
- [ ] As required by the reviewer, all functions ask the user an input should be in the class for getting user data. Which in your case, Run. Please clear your App class as per the single responsibility rule.